### PR TITLE
Rhys/no preditor

### PR DESCRIPTION
**Pull request by wheresrhys**
Moved markets onto a capi list, and technology onto a capi concept stream

### CODE REVIEW

File: server/graphql/apis/markets.js [hunk 1]
```diff
@@ -5,10 +5,10 @@ import ids from '../../config/ids';
...
 
 export default () => ({
-	type: types.page,
+	type: types.list,
 	resolve: () => {
 		metrics.count('graphql.apis.markets.count', 1);
-
-		return backends.capi.page(ids.pages.markets);
+		return backends.capi.list(ids.lists.markets)
+			.then(result => Object.assign({}, result, {title: 'markets'}))
```
Comment (ironsidevsquincy) [inline server/graphql/apis/markets.js]: should we override this? they can add a title to the list, which we should probably respect

File: server/graphql/apis/technology.js [hunk 2]
```diff
@@ -1,14 +1,37 @@
...
+			type: graphql.GraphQLInt,
+			defaultValue: 0
+		},
+		limit: {
+			type: graphql.GraphQLInt,
+			defaultValue: 20
+		}
+	},
+	resolve: (root, { from, limit }) => {
+		metrics.count('graphql.apis.fast-ft.count', 1);
```
Comment (ironsidevsquincy) [inline server/graphql/apis/technology.js]: `technology`

File: server/graphql/apis/markets.js (hunk 1 above)
Comment (wheresrhys) [inline server/graphql/apis/markets.js]: In principle I think we shouldn't, but in the interest of this not changing the current behaviour I'd like to keep the title the same. I don't want to accidentally change the subheading on the home page and then be chasing that bug.

Comment (ironsidevsquincy) [inline server/graphql/apis/markets.js]: 👍

Comment (ironsidevsquincy) [inline server/graphql/apis/markets.js]: (Though we should get Mus/someone to update the list's title)

**Pull request merged**